go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/zyedidia/clipper v0.1.1
 	github.com/zyedidia/glob v0.0.0-20170209203856-dd4023a66dc3
 	github.com/zyedidia/json5 v0.0.0-20200102012142-2da050b1a98d
-	github.com/zyedidia/tcell/v2 v2.0.10 // indirect
+	github.com/zyedidia/tcell/v2 v2.0.10
```

Comment:
Slightly unrelated to the PR, right?

Reply:
I don't think I made this change.  I think the build did.  Does it have an effect?

Reply:
I suggest to stay with the current content, as long as it isn't really needed here.

@dmaluka:
What's your opinion about it, since you're more familiar in `go` than me.

Reply:
I'm not really familiar with this particular aspect of Go. But I agree it doesn't look like we should change this tcell line.